replicatedhq/ship
# Maintain helm values keys order

What I Did
------------
Ship init Helm values now appear in order after saving and navigating back to the /values page.

How I Did it
------------
Merging helm values from the upstream and the ship state now uses yaml.MapSlice to maintain order of keys.

How to verify it
------------
Save the /values page and then navigate back. Keys should appear in order.

Description for the Changelog
------------
Ship init helm value keys will now maintain order after saving and returning to the /values page in the nav cycle.


Picture of a Boat (not required but encouraged)
------------
